manifest.php

```diff
@@ -27,11 +27,11 @@
     'label' => 'extension-tao-testqti-previewer',
     'description' => 'extension that provides QTI test previewer',
     'license'     => 'GPL-2.0',
-    'version' => '2.15.3',
+    'version' => '2.15.4',
```

Comment:
```suggestion
    'version' => '2.16.0',
```
Because deps were bumped.